.github/workflows/ci-pr-docs.yml

```diff
@@ -0,0 +1,52 @@
+name: ci pr docs
+env:
+  VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+  VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID }}
+on:
+  # TODO: change to pull_request when docusaurus site is ready
+  # pull_request:
+  push:
+    branches:
+      - idea404/docusaurus-docs
```

Comment:
can we give this a test run by opening a PR to the `idea404/docusaurus-docs` branch, and seeing if the deployment & comment works?

Reply:
alternatively, we could say on push: `ideo404/ci-docs` so that it should populate here directly